libs/src/services/schemaValidator/schemas/trackSchema.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
We definitely should have a single source of truth for these. Can we pull this out to some common location?

Reply:
i was thinking about how to do this in a lightweight. i tried to avoid putting the onus on build scripts to pull these into the appropriate locations during npm builds or docker builds. any ideas here?

Reply:
I think they should live in a separate repo or separate folder-- perhaps as a separate repo we could have exported python packages & npm packages and then it could just be installed with pip or NPM

Reply:
@raymondjacobson  ideally yes, but do we need to do that now? i was thinking if we wanted the overhead of managing a pip package now on top of everything else

Reply:
for now, just to get this out the door with as little overhead as possible i'd actually prefer making a dedicated build step to pull the schema from a separate folder in this repo like top level /schemas or something